app/konnect/getting-started/configure-data-plane-node.md

```diff
@@ -1,22 +1,22 @@
 ---
-title: Configure a Runtime
+title: Configure a Data Plane Node
```

Comment:
❗ For reviewers: Here and throughout this PR, if you could make sure I used the correct words in place of "runtime" that would be great. I sometimes had a hard time deciding if it should be control plane, data plane node, or data plane.